bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -35,6 +35,52 @@ pub struct Room {
     timeline: RwLock<Option<Arc<Timeline>>>,
 }
 
+pub enum MembershipState {
```

Comment:
This can use `#[derive(uniffi::Enum)]` just like the `Membership` enum above, no? This way the `udl` file doesn't need a matching entry.